lib/internal/bootstrap_node.js

```diff
@@ -14,9 +14,7 @@
     process._eventsCount = 0;
 
     const origProcProto = Object.getPrototypeOf(process);
-    Object.setPrototypeOf(process, Object.create(EventEmitter.prototype, {
-      constructor: Object.getOwnPropertyDescriptor(origProcProto, 'constructor')
-    }));
+    Object.setPrototypeOf(Object.getPrototypeOf(process), EventEmitter.prototype);
```

Comment:
Gah, that's what I get for writing the same thing on a few machines to test it all out. Fixed.